app/uk/gov/hmrc/gform/sharedmodel/formtemplate/EmailParameter.scala

```diff
@@ -18,20 +18,28 @@ package uk.gov.hmrc.gform.sharedmodel.formtemplate
 
 import play.api.libs.functional.syntax._
 import play.api.libs.json._
-case class EmailParameter(emailTemplateVariable: String, value: FormCtx)
 
+case class EmailParameter(emailTemplateVariable: String, value: TextExpression)
```

Comment:
Please do not use `TextExpression`, use `Expr` instead. 

(`TextExpression` is supposed to be used for one of the possible result of parsing "value" json field.)